metadata/top-resources.yml

```diff
@@ -0,0 +1,263 @@
+description: |
```

Comment:
Is this file used in this PR? I can't see where it is referenced.

Reply:
I've used it in the script that extracts examples for top N resources.